addons/account/models/account_journal.py

```diff
@@ -662,7 +662,9 @@ def create(self, vals_list):
             # Create the bank_account_id if necessary
             if journal.type == 'bank' and not journal.bank_account_id and vals.get('bank_acc_number'):
                 journal.set_bank_account(vals.get('bank_acc_number'), vals.get('bank_id'))
-
+        for journal in journals:
+            if journal.restrict_mode_hash_table and not journal.secure_sequence_id:
+                journal._create_secure_sequence(['secure_sequence_id'])
```

Comment:
```suggestion
            if journal.restrict_mode_hash_table and not journal.secure_sequence_id:
                journal._create_secure_sequence(['secure_sequence_id'])
```
Why loop twice on `journals`? :wink: